build/android/app/build.gradle

```diff
@@ -0,0 +1,106 @@
+apply plugin: 'com.android.application'
+android {
+	compileSdkVersion 29
+	buildToolsVersion '29.0.2'
```

Comment:
Specifying a verified version is important in Android. A new version of the SDK may suddenly break everything. It must be saved!

Reply:
Since two major releases of Android, Google remove the need of it. 
I at least develop for the present, not for the future. Doing it for the future, you will never be finished, and nobody knows about the future

Reply:
Okay, i removed this lines.
`Checking the license for package Android SDK Build-Tools 28.0.3 in /Users/Admin/Library/Android/sdk/licenses
License for package Android SDK Build-Tools 28.0.3 accepted.
Preparing "Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)".
"Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" ready.
Installing Android SDK Build-Tools 28.0.3 in /Users/Admin/Library/Android/sdk/build-tools/28.0.3
"Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" complete.
"Install Android SDK Build-Tools 28.0.3 (revision: 28.0.3)" finished.`